content/en/docs/v3/getting-started/_index.md

```diff
@@ -10,11 +10,11 @@ weight: 10
 
 Jenkins X 3.x alpha includes a new install approach which uses tools like helm 3, kpt, kustomize and includes a number of [improvements](/docs/v3/about/benefits/) over Jenkins X 2.
 
-- Download the new `jx` CLI https://github.com/jenkins-x/jx-cli/releases/latest
+- Download the new `jx` CLI and install it in your executable path by running the appropriate bash script for your system given here: https://github.com/jenkins-x/jx-cli/releases/latest   
```

Comment:
Seems like there is some white space after the sentence (after latest)

Reply:
I'm all for hooks to ensure basic formatting conventions (whatever we choose them to be). :)